robot_framework/config.py

```diff
@@ -6,22 +6,60 @@
 # Whether the robot should be marked as failed if MAX_RETRY_COUNT is reached.
 FAIL_ROBOT_ON_TOO_MANY_ERRORS = True
 
-# Error screenshot config
+# Process report email
 SMTP_SERVER = "smtp.aarhuskommune.local"
 SMTP_PORT = 25
-SCREENSHOT_SENDER = "robot@friend.dk"
+STATUS_SENDER = "itk-rpa@mkb.aarhus.dk"
+
+# Email texts
+SUBJECT_BASE = "Robotstatus for Masseoprettelse i KMD Nova: "
```

Comment:
This is a little weird...

Reply:
It is very weird. You're weird! I'll sleep on it...